### PR TITLE
Adapter testing

### PR DESCRIPTION
**Pull request by Matt-Blanco**
Fixed up all of the adapters to work properly on the same set of test charts. There were several bugs in the Observable Adapter where even though the trees were being generated they did not have the proper data, or null values were present in the descriptions. Additionally faceted charts were not being properly parsed in the Vega Adapter.

### CODE REVIEW

File: packages/adapters/src/VegaAdapter.ts [hunk 1]
```diff
@@ -12,16 +12,16 @@ let spec: Spec;
...
+    view = visObject;
+    spec = helperVisInformation;
+    if (view.items.some((el: any) => el.role === "scope")) {
+        return parseFacets();
+    } else {
+        return parseSingleChart(view);
     }
+}
 
-function parseMultiViewChart(): NestedChart {
```
Comment (jonathanzong) [inline packages/adapters/src/VegaAdapter.ts:24]: great -- i had a hunch that the NestedChart type might not end up being used. now that there's no longer any references to this type, can we remove it?

File: packages/adapters/src/ObservablePlotAdapter.ts [hunk 2]
```diff
@@ -169,18 +172,19 @@ function parseLegend(plot: any, svg: Element): Legend { //TODO: Does not support
             if (isNaN(parseInt(c.textContent!))) {
                 a.push(c.textContent as never);
             } else {
-                a.push(parseInt(c.textContent!) as never);
+                a.push(parseInt(c.textContent!.replace(/,/g, '')) as never);
```
Comment (jonathanzong) [inline packages/adapters/src/ObservablePlotAdapter.ts]: can we leave a comment documenting the use of `as never` here?

File: packages/adapters/src/Types.ts [hunk 3]
```diff
@@ -68,7 +68,7 @@ export const nestedChart = (fields: Omit<NestedChart, 'type'>): NestedChart => {
  *   field: the object field that will be used to compare data values to range values
  */
 export type Guide = {
-    values: string[] | number[]
+    values: string[] | number[],
     title: string
     data: any[]
```
Comment (jonathanzong) [inline packages/adapters/src/Types.ts]: nitpick: add commas to the end of lines consistently

File: packages/adapters/src/VegaAdapter.ts (hunk 1 above)
Comment (Matt-Blanco) [reply packages/adapters/src/VegaAdapter.ts:24]: Yeah just added a commit removing the `nestedChart ` type. While it was helpful in theory, if a faceted chart exists there should always be a way to get the values each nested chart is faceted by

**Pull request merged**